modules/rest-api/pages/rest-xdcr-create-replication.adoc

```diff
@@ -82,6 +83,11 @@ The value can be `High`, `Medium`, or `Low`.
 The default value is `High`.
 For information, see xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-priority[XDCR Priority].
 
+The `network_type` flag determines whether _internal_ (default) or _alternate_ IP addresses are used when communicating with the target cluster.
```

Comment:
Do we need to say "optional" somewhere? Similar to "filterExpression" above?

Reply:
I've added.